sparrow-py/pysrc/sparrow_py/_result.py

```diff
@@ -1,2 +1,59 @@
+from . import _ffi
+import pandas as pd
+from typing import Iterator
+import pyarrow as pa
+
 class Result(object):
     """Result of running a timestream query."""
+
+    def __init__(self, ffi_execution: _ffi.Execution) -> None:
+        """Create the result."""
+        self._ffi_execution = ffi_execution
+
+    def to_pandas(self) -> pd.DataFrame:
```

Comment:
One annoying thing is that you can't do:

```
results = expr.run()
results.to_pandas()
results.to_pandas()
```

Because the first call "consumes" the results. We may want to make this actually capture the expression and execute it on demand, which may change some of the naming. Let's discuss tomorrow?

@kerinin